sidebars.js

```diff
@@ -1,119 +1,119 @@
 module.exports = {
```

Comment:
Please revert the quotes changes to this file, standard Docusaurous convention is single quotes.